src/web/nextui/src/utils.ts

```diff
@@ -0,0 +1,127 @@
+import { Assertion, AssertionType, CsvRow, TestCase } from "@/../../../types";
```

Comment:
@typpo I noticed the web UI doesn't have a dependency on the library itself (except for types), so I duplicated what I needed here, but i guess reusing the library directly would be better.